llvm/test/CodeGen/AMDGPU/no-fold-accvgpr-mov.ll

```diff
@@ -0,0 +1,124 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -O3 -mtriple=amdgcn--amdhsa -mcpu=gfx942 %s -o - | FileCheck %s --check-prefixes=GFX942
+; RUN: llc -O3 -mtriple=amdgcn--amdhsa -mcpu=gfx908 %s -o - | FileCheck %s --check-prefixes=GFX908
+
+define amdgpu_kernel void @test(i1 %arg, i1 %arg1) {
```

Comment:
i1 kernel arguments are a kind of special case, can you replace these with i32 and explicit compares to 0?